src/i18n.rs

```diff
@@ -107,6 +107,42 @@ impl Default for TeamHelper {
     }
 }
 
+enum TeamHelperParam {
+    /// `{{team-text team name}}`
+    Name,
+
+    /// `{{team-text team description}}`
+    Description,
+
+    /// `{{team-text team role (lookup member.roles 0)}}`
+    Role(String),
+}
+
+impl TeamHelperParam {
+    fn fluent_id(&self, team_name: &str) -> String {
+        match self {
+            TeamHelperParam::Name => format!("governance-team-{team_name}-name"),
+            TeamHelperParam::Description => format!("governance-team-{team_name}-description"),
+            TeamHelperParam::Role(role_id) => format!("governance-role-{role_id}"),
+        }
+    }
+
+    fn english<'a>(&self, team: &'a serde_json::Value) -> &'a str {
+        match self {
+            TeamHelperParam::Name => team["website_data"]["name"].as_str().unwrap(),
+            TeamHelperParam::Description => team["website_data"]["description"].as_str().unwrap(),
+            TeamHelperParam::Role(role_id) => {
+                for role in team["roles"].as_array().unwrap() {
+                    if role["id"] == *role_id {
+                        return role["description"].as_str().unwrap();
+                    }
+                }
+                panic!("no role {:?}", role_id);
```

Comment:
issue: let's fail gracefully; I don't want the website to crash whenever the teams repo is updated

I'd just have it output the role ID

Reply:
I don't think this would crash on a team repo update &mdash; the JSON looks like this:

```js
{
  "name": "spec",
  "members": [
    ...
    {
      "name": "Joel Marcey",
      "github": "JoelMarcey",
      "roles": [
        "spec-editor"
      ]
    }
  ],
  "website_data": {
    "name": "Specification team",
    "description": "Creating and maintaining the specification for the Rust language",
    "page": "spec",
    ...
  },
  "roles": [
    {
      "id": "spec-editor",
      "description": "Editor"
    }
  ]
}
```

where `.members.*.roles.*` are guaranteed to line up with exactly one `.roles.*.id`.

There is a test here to enforce it:

https://github.com/rust-lang/team/blob/ae0fe5cb83bbb24c9533c6829f2ca11667e8ab2d/src/validate.rs#L840-L849

The static-api teams.json cannot be published without that validation passing.

We can make this more robust to broken teams.json but all the other unwraps in this code would also be crashable in that situation.

Reply:
Changed to return role_id, with a comment saying it should be unreachable.